contributing_to_docs/doc_guidelines.adoc

```diff
@@ -236,16 +242,16 @@ $ ln -s <target_directory> <link_name>
 For example, if you are creating the links in a directory that is two levels deep, such as `cli_reference/openshift_cli`, use the following commands:
 +
 ----
-$ ln -s ../../images/ images
-$ ln -s ../../modules/ modules
-$ ln -s ../../snippets/ snippets
-$ ln -s ../../_attributes/ _attributes
+[openshift-docs ~]$ cd cli_reference/openshift_cli
+[openshift_cli ~]$ ln -s ../../images/ images
+[openshift_cli ~]$ ln -s ../../modules/ modules
+[openshift_cli ~]$ ln -s ../../snippets/ snippets
+[openshift_cli ~]$ ln -s ../../_attributes/ _attributes
 ----
 +
 Be sure to adjust the number of levels to back up (`../`) depending on how deep your directory is.
 
-If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`.
-====
+If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`. For example, to unlink the `images-typo` symlink in the `cli_reference/openshift_cli` directory, run `unlink images-typo`.
```

Comment:
```suggestion
If you accidentally create an incorrect link, you can remove that link by using `unlink <link_name>`. For example, to unlink the `images` symlink in the `cli_reference/openshift_cli` directory, run `unlink images`.
```
Not sure what the "typo" meant here.

Reply:
I was doing that to show that you made a mistake in creating the link. Could be in the name of the symlink or could be in the directory path you chose.

But if what I have here is more confusing than helpful, I'll remove it